spring-boot-project/spring-boot-test/pom.xml

```diff
@@ -196,6 +211,38 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
```

Comment:
I am confused. In the `spring-boot` module you've set the maven compiler phases to `none`. Why isn't it necessary here?

Reply:
Good catch, it is, I will update the PR accordingly.